src/js/media-controller.js

```diff
@@ -406,16 +341,11 @@ class MediaController extends MediaContainer {
     if (index > -1) return;
 
     els.push(el);
-
-    Object.keys(MediaUIStates).forEach((stateConstName)=>{
-      const stateDetails = MediaUIStates[stateConstName];
-
-      propagateMediaState(
-        [el],
-        MediaUIProps[stateConstName],
-        stateDetails.get(this)
-      );
-    });
+    if (this.#mediaStore) {
```

Comment:
```suggestion

    if (this.#mediaStore) {
```

a little white space helps with code readability